### DISCUSSION TITLE
Application Demo based on NVIDIA Jetson supported models(5) 基于Jetson 模型开发应用 Demo(5)

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#627] [Other]
# Application Demo based on NVIDIA Jetson supported models(5) 
 基于Jetson 模型开发应用 Demo(5) 

Develop based on one of the following models:
 
 [MMPreTrain](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmpretrain.html): mobilenetv2, mobileone, 
 
 [MMDetection](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet.html): rtmdet, yolov3, yolox
 
 [MMSegmentation](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmseg.html): fastscnn, bisenetv2, stdc
 
 [MMPose](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmpose.html): rtmpose, simcc, yolox-pose
  
  Project requirements:
  1. Submit the project's GitHub/Hackster project link.
  2. Provide a deployment tutorial that complies with the code standards.
  3. Create a demo video showcasing the application.

请从以下模型中选择一个进行开发：
 
 [预训练模型列表](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmpretrain.html ): mobilenetv2, mobileone, efficientnet 
 
 [目标检测模型列表](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet.html#supported-models): rtmdet, yolov3, yolox mmseg: fastscnn, bisenetv2, stdc 
 
 [姿态估计模型列表](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmpose.html): rtmpose, simcc,yolox-pose
 
 [语义分割模型列表](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmseg.html): fastscnn, bisenetv2, stdc 
 
 项目要求： 
 1. 提交项目Github项目链接 
 2. 产出部署教程，符合代码规范 
 3. 结果展示：创建一个演示应用程序，展示调优后的模型在 Jetson 平台上的性能和效果。制作演示视频，展示模型的输入、输出和关键功能。
 
 硬件的使用可通过[部署平台](https://platform.openmmlab.com/deploee?lang=zh-CN)，使用Jetson Orin

## Task Information

**Repository**：Other

**Difficulty**：Medium

**Points**：40

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：yinfan98

**Claim Time**：2023年08月14日

**PR Link**：https://github.com/yinfan98/mmpretrain_Jetson

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by FlamingoPg** [comment]
finish here and my repo below~
https://github.com/yinfan98/mmpretrain_Jetson
